CSS/index.css

```diff
@@ -106,4 +111,103 @@ input#menuButton {
     transform: translateY(-305px);
     transition: transform 500ms;
   }
+
+  #bGImage {
+    margin-top: -213px;
+  }
+}
+/* Main Banner */
+#bGImage {
+  background-image: url("../assets/BGimage.png");
+  background-color: rgb(241, 241, 241);
+  height: 700px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  z-index: -1;
+
+  /* is this negative margin ok? */
+}
+
+.b_g_text {
+  text-align: center;
+  position: absolute;
+  font-size: 40px;
+  font-family: Arial, Helvetica, sans-serif;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  color: white;
+}
+
+h1 {
+  display: flex;
+  flex-direction: row;
+}
+
+h2 {
+  font-size: 80px;
+}
+.b_g_text_two {
+  left: 30%;
+  transform: translate(20%, -50%);
+}
+#spanOne {
+  color: #000000;
+}
+
+#spanTwo {
+  color: #000000;
+}
+
+/* page slider */
+
+#pageSliderContainer {
+  width: 100%;
+  height: 10%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+#pageSlider {
+  border: 1px solid black;
+  border-radius: 50%;
+  width: 6%;
+  min-width: 50px;
+  max-width: 10px;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  cursor: pointer;
+  position: relative;
+  z-index: 1;
+  /* content: "/9679"; */
+  /* tried it insert circle via unicode in CSS, why didn't
+  this work? */
+}
+
+#pointer {
+  font-size: 220%;
+  margin-top: 5px;
+}
+
+@media only screen and (max-width: 550px) {
+  .b_g_text {
+    transform: translate(-50%, 50%);
+  }
+
+  .b_g_text_two {
+    color: black;
```

Comment:
color: black is ok for quick development but I want us to use real hex values or rgb() for colors on a real project like this